docs/user/dmca/archive/2022-06-07.rst

```diff
@@ -0,0 +1,42 @@
+:nosearch:
+
+2022-06-07
+==========
+
+.. note::
+    The project maintainer was notified about this report and instructed to
+    submit a counter if they believed this request was invalid. The user removed
+    the project manually, and no further action was required.
+
+Are you the copyright owner or authorized to act on the copyright owner's behalf?
+    Yes
+
+What work was allegedly infringed? If possible, please provide a URL:
+    https://www.dicomstandard.org/current 
+
+What files or project should be taken down? You should list URLs to each piece of infringing content. If you allege that the entire project is infringing on copyrights you hold, please specify the entire project as infringing:
+    https://dicom-standard.readthedocs.io/en/latest/index.html
+
+Is the work licensed under an open source license?
+    No 
+
+What would be the best solution for the alleged infringement?
+    Complete Removal.
+
+Do you have the alleged infringer's contact information? Yes. If so, please provide it:
+    [private]
+
+Type (or copy and paste) the following statement: "I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration."
+    I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration.
+
+Type (or copy and paste) the following statement: "I swear, under penalty of perjury, that the information in this notification is accurate and that I am the copyright owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed."
+    I swear, under penalty of perjury, that the information in this notification is accurate and that I am the copyright owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed.
+
+Please confirm that you have you have read our Takedown Policy: https://docs.readthedocs.io/en/latest/dmca/index.html
```

Comment:
```suggestion
Please confirm that you have read our Takedown Policy: https://docs.readthedocs.io/en/latest/dmca/index.html
```

Reply:
This is actually a problem with our template, I'll fix this there. This is the file that the user sent us